package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "bindings": "1.2.1",
-    "nan": "2.0.7"
+    "nan": "^2.0.7"
```

Comment:
Agree, but this is kind of an emergency with the Node6 release.  This could go to "2.3.2" if that makes everyone feel better.  If this is holding up cutting a release, I can send in a PR.  I've got a bunch of code that is currently broken awaiting a release, so I'm motivated to help. :)